config/index.js

```diff
@@ -74,12 +74,12 @@ const locals = {
 		{
 			"title": "More on the way",
 			"image": "features/coming-soon.png",
-			"meta": "50% faster burns, simultaneous writing for multiple drives.<br/><a target='_blank' href='https://github.com/resin-io/etcher/milestones'>View our roadmap</a>"
+			"meta": "50% faster flashes, simultaneous writing for multiple drives.<br/><a target='_blank' href='https://github.com/resin-io/etcher/milestones'>View our roadmap</a>"
 		}
 	],
   story: [
-		"Here at <a target='_blank' href='https://resin.io'>resin.io</a> we have thousands of users working through our getting started process and until recently we were embarassed about the steps that involved burning an SD card. There was a separate track for each Mac/Windows/Linux and several manual and error-prone steps along the way.",
-		"To our surprise there was nothing out there that fitted our needs. So we built Etcher, a SD card burner app that is simple for end users, extensible for developers, and works on any platform. "
+		"Here at <a target='_blank' href='https://resin.io'>resin.io</a> we have thousands of users working through our getting started process and until recently we were embarassed about the steps that involved flashing an SD card. There was a separate track for each Mac/Windows/Linux and several manual and error-prone steps along the way.",
+		"To our surprise there was nothing out there that fitted our needs. So we built Etcher, a SD card flasher app that is simple for end users, extensible for developers, and works on any platform. "
```

Comment:
I know this doesn't specifically pertain to this PR, but there seems to be some inconsistencies with "a SD" and "an SD", with the latter sounding better.